cloudbuild.yaml

```diff
@@ -24,6 +24,5 @@ steps:
       - '--tag=gcr.io/$PROJECT_ID/gcp-variant-transforms:${_CUSTOM_TAG_NAME}'
       - '--file=docker/Dockerfile'
       - '.'
-    dir: '.'
```

Comment:
Hmm, so it's not needed? I remember we had to add it but don't remember why? :)

Reply:
Correct, AIUI, dir is the path relative to /workspace where everything runs. '.' is the default value.